### PR TITLE
Problem: generated classes are outdated

### PR DESCRIPTION
**Pull request by mehdisadeghi**
Solution: regenerate them.

I had to add `MATERIAL_ID` manually to `keywords.py` and moreover I had to change all `shape=[]` and `shape=None`s to `shape=[1]` to make it work.

### CODE REVIEW

**Comment by mehdisadeghi** [comment]
@tuopuu You can use this branch for your YAML tests.

**Comment by tuopuu** [comment]
Thanks @mehdisadeghi, I'll merge this branch locally to my work branch.

File: simphony/core/keywords.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
         name='ElectrostaticField',
         definition='Electrostatic field',  # noqa
         key='ELECTROSTATIC_FIELD',
-        shape=[],
+        shape=[1],
```
Comment (tuopuu) [inline simphony/core/keywords.py]: Shape of the field should be [3]. This causes an error when instantiating ConstantElectrostaticFieldModel.

Comment (mehdisadeghi) [inline simphony/core/keywords.py]: @tuopuu we have to apply it to the metadata. Then I'll update this branch. First task for tomorrow.

Comment (mehdisadeghi) [inline simphony/core/keywords.py]: @tuopuu see this PR please: https://github.com/simphony/simphony-metadata/pull/48

**Pull request merged**